ca/certificate-authority_test.go

```diff
@@ -38,12 +38,6 @@ var (
 
 	// CSR generated by Go:
 	// * Random public key
-	// * CN = not-example.com
-	// * DNSNames = [none]
-	NoSANCSR = mustRead("./testdata/no_san.der.csr")
```

Comment:
These are replicated in `csr/csr_test.go` but are somewhat simplified to
not need an entire CSR for each test.

On Wed, May 25, 2016 at 1:17 PM Jacob Hoffman-Andrews <
notifications@github.com> wrote:

> In ca/certificate-authority_test.go
> <https://github.com/letsencrypt/boulder/pull/1826#discussion_r64645488>:
>
> > @@ -38,12 +38,6 @@ var (
> >
> >  	// CSR generated by Go:
> >  	// * Random public key
> > -	// * CN = not-example.com
> > -	// * DNSNames = [none]
> > -	NoSANCSR = mustRead("./testdata/no_san.der.csr")
>
> Where did these test cases go?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1826/files/b7f1b52ba41c8ef7bf11ec21ed75fb96a8be4fdc#r64645488>
>